### PR TITLE
Introduce filtering query list function

### LINKED ISSUE #248: [feature request] Filter query list by query title
# What?

- Add "filter query list field"

Default screen
Frame 3011

User can filter query list by title 
Frame 3014

# Value
When user wants to find the query that he wrote in the past, user can quickly find it by filtering the list.

# Spec
- Show filter query field above the query list
- When user user input something in the filter field.. 
  - Filter query list by title ~~or content~~
  - Show un-filter button
  - If no content is found, show "no query found" in the query list.
- When user puts the un-filter button..
  -  Filter field goes back to default state
  -  Un-filter query list.

### PR DESCRIPTION
**Pull request by hashcc**
Close #248 

Created working code of #248

Frame 3016

@hokaccha @morishin 
Would you try this code?
And let me know what you think. (Whether valuable to merge or not)

### CODE REVIEW

File: src/renderer/components/QueryList/QueryList.css [hunk 1]
```diff
@@ -18,6 +21,31 @@
   cursor: pointer;
 }
 
+.QueryList-new .QueryList-filter {
```
Comment (hashcc) [inline src/renderer/components/QueryList/QueryList.css:24]: UI style is copied from "Data source" filtering field 😀
Group 2809

File: src/renderer/components/QueryList/QueryList.tsx [hunk 2]
```diff
@@ -47,28 +47,36 @@ const QueryList: React.FC<Props> = ({
...
-        {query.title}
-      </li>
-    );
-  });
+  const [filterText, setFilterText] = useState("");
+  const handleChange = (event) => {
+    setFilterText(event.target.value);
+  };
+
+  console.log(queries);
```
Comment (morishin) [inline src/renderer/components/QueryList/QueryList.tsx:55]:
```suggestion
```

**Review by morishin — APPROVED** [review]
LGTM!

**Comment by morishin** [comment]
Thanks @hashcc ! I'll merge it.🚀

**Pull request merged**